src/java/main/org/apache/zookeeper/version/util/VerGen.java

```diff
@@ -93,6 +93,14 @@ public static void generateFile(File outputDir, Version version, int rev, String
         } catch (IOException e) {
             System.out.println("Unable to generate version.Info file: "
                     + e.getMessage());
+            if (w != null) {
```

Comment:
wow this is a weird one... we have it in the finally... try with resources would make this a bit nicer too...